### PR TITLE
Cars counter (dashboard + fixes)

### PR DESCRIPTION
**Pull request by yprez**
Add cars counter to homepage dashboard (#793)

### CODE REVIEW

File: routes/api-18/controllers/users.controller.js [hunk 1]
```diff
@@ -149,39 +149,38 @@ class UsersController {
...
-                                stat.last_1h_entrance = result[0][0]['last_1h_entrance'];
-                                stat.last_1h_exit = result[0][0]['last_1h_exit'];
-                            }
-                            res.status(200).json({groups: groups, stats: stat});
 
-                        });
-                    });
+                let stat = {};
+
+                const groupStats = await knex.raw(query);
```
Comment (yprez) [inline routes/api-18/controllers/users.controller.js:155]: thanks @NitayRabi !

File: routes/api-18/controllers/users.controller.js [hunk 2]
```diff
@@ -149,39 +149,53 @@ class UsersController {
...
+                    stat.ticketing = ticketStats[0][0]['ticketing'];
+                    stat.last_24h_first_entrance = ticketStats[0][0]['last_24h_first_entrance'];
+                    stat.last_1h_first_entrance = ticketStats[0][0]['last_1h_first_entrance'];
+                    stat.last_24h_entrance = ticketStats[0][0]['last_24h_entrance'];
+                    stat.last_24h_exit = ticketStats[0][0]['last_24h_exit'];
+                    stat.last_1h_entrance = ticketStats[0][0]['last_1h_entrance'];
+                    stat.last_1h_exit = ticketStats[0][0]['last_1h_exit'];
                 }
+
+                // TODO: deduplicate code with api_gate_routes
```
Comment (yprez) [inline routes/api-18/controllers/users.controller.js:183]: @amotenko optimally this should be taken out to a model / helper function. Maybe take out the "stats" to an api endpoint.

copy-pasta was the quickest. Improve, or merge like this?

Comment (amotenko) [reply routes/api-18/controllers/users.controller.js:183]: merge first, then improve.
You have all the time in the world!
Until code freeze, that is

Comment (yprez) [reply routes/api-18/controllers/users.controller.js:183]: :+1:

**Pull request merged**